go.mod

```diff
@@ -56,6 +56,7 @@ require (
 	github.com/kballard/go-shellquote v0.0.0-20180428030007-95032a82bc51
 	github.com/libvirt/libvirt-go v3.4.0+incompatible
 	github.com/machine-drivers/docker-machine-driver-vmware v0.1.1
+	github.com/machine-drivers/machine v0.16.2 // indirect
```

Comment:
@hetong07 
how about only keeping github.com/machine-drivers/machine

and then removing the 
```
github.com/docker/machine
```

(and ensure u rebase from upstream and `go mod tidy`)